tests/behavior/behavior.rs

```diff
@@ -139,7 +140,7 @@ macro_rules! behavior_test {
                         let root = format!("{}{}/", root, uuid::Uuid::new_v4());
                         cfg.insert("root".into(), root);
 
-                        let op = Operator::from_iter($service, cfg.into_iter())?;
+                        let op = Operator::from_iter($service, cfg.into_iter())?.layer(LoggingLayer);
```

Comment:
```suggestion
                        let op = Operator::from_iter($service, cfg.into_iter())?
                        .layer(LoggingLayer);
```
make this `?` more obvious.

Reply:
Not needed so far, I guess